src/python/pants/backend/python/util_rules/package_dists_test.py

```diff
@@ -1438,12 +1436,7 @@ def test_no_dist_type_selected() -> None:
         rule_runner.request(
             BuiltPackage,
             inputs=[
-                PythonDistributionFieldSet(
-                    address=address,
-                    provides=PythonProvidesField(
-                        PythonArtifact(name="aaa", version="2.2.2"), address
```

Comment:
This duplication with the content of the test BUILD file above is a code smell that supports this change.